CoreStubs/ArchiveControl/module.php

```diff
@@ -54,9 +80,27 @@ public function DeleteVariableData(int $VariableID, int $StartTime, int $EndTime
         throw new Exception('Not implemented');
     }
 
-    public function GetAggregatedValues(int $VariableID, int $AggregationLevel, int $StartTime, $EndTime, $Limit)
+    public function GetAggregatedValues(int $VariableID, int $AggregationSpan, int $StartTime, int $EndTime, int $Limit)
     {
-        throw new Exception('Not implemented');
+        if (empty($this->Archive[$VariableID]['AggregatedValues'])) {
```

Comment:
Der Check funktioniert so nicht mehr.
1. Wenn die Variablendaten noch nicht angelegt wurden, dann fliegt dir das um die Ohren
2. Wenn die Variablendaten schon da sind, dann ist das Array nicht leer, da ja 7 leere Arrays darin sind

Reply:
if (empty($this->Archive[$VariableID]['AggregatedValues'][$AggregationSpan])) {

Das sollte klappen.